min-turnup/phase1/azure/configure-vm.sh

```diff
@@ -0,0 +1,69 @@
+#! /bin/bash
+
+set -x
+set -o errexit
+set -o pipefail
+set -o nounset
+
+mkdir -p /etc/systemd/system/docker.service.d/
+cat <<EOF > /etc/systemd/system/docker.service.d/clear_mount_propagtion_flags.conf
+[Service]
+MountFlags=shared
+EOF
+cat <<EOF > /etc/systemd/system/docker.service.d/overlay.conf
+[Service]
+ExecStart=
```

Comment:
This resets the previous value. Well, it's a bit of a special case for `ExecStart` and certain other attributes, but that's what you have to do and it complains otherwise.